scripts/pipeline/set_environment_variables/set_slack_notify_production_release.sh

```diff
@@ -1,10 +1,10 @@
 #! /bin/bash
     echo "COMMIT_MESSAGE:"
 
-    echo ${COMMIT_MESSAGE}
```

Comment:
So just looking into [brace-expansion](https://ss64.com/bash/syntax-expand.html) as I wasn't entirely sure on the difference. 
Are we sure it's not another character that's not escaped correctly causing this? 
As for looking at `{}` using this should have been fine, maybe in conjunction with the double quotes, but for the sanitised message it wouldn't have made a difference.

Reply:
Taking the braces out seems to fix the issue, it seems to treat some new lines as an overwrite of the message. it's something to do with the braces and the interaction with the shell.

Reply:
best to take the braces out as then it won't perform expansion in any case.

Reply:
Ah, we've found another bug then 🙃